doc/reference/tooling/tt_cli/import.rst

```diff
@@ -278,6 +363,10 @@ Options
 
 ..  option:: --rollback-on-error
 
-    Applicable only when ``crud`` is used.
+    **Applicable to:** ``tt crud import``
 
     Specify whether any operation failed on a router leads to rollback on a storage where the operation is failed.
```

Comment:
Even though it's not the part of this PR, this line is misleading. This flag enables the rollback per-storage. So, if an error had happened on storage, **only this storage** will be rolled back. Any operation failed on a router does not trigger any roll back on a storage. Though, in 99% of cases, if an error had happened on a router, it had happened before doing any operations on storages.